go1.14/amd64/rootfs/compilers.yaml

```diff
@@ -0,0 +1,25 @@
+---
+
+windows:
+  386:
+    CC:  i686-w64-mingw32-gcc
+    CXX: i686-w64-mingw32-g++
+  amd64:
+    CC:  x86_64-w64-mingw32-gcc
+    CXX: x86_64-w64-mingw32-g++
+
+darwin:
```

Comment:
darwin probably shouldn't have be in here since those are in the dedicated `darwin` image (perhaps I had this wrong in the `main` compilers.yaml).